porcupine/plugins/run/dialog.py

```diff
@@ -58,7 +58,7 @@ def validate(self, *junk_from_var_trace: object) -> None:
 
 class _CommandAsker:
     def __init__(self, ctx: common.Context):
-        self.window = tkinter.Toplevel()
+        self.window = tkinter.Toplevel(name="command_asker")
```

Comment:
I think `run_command_asker` would be a better name, just in case another plugin decides to also make a widget named `command_asker`.